src/lib/modules/blockchain_process/proxy.js

```diff
@@ -13,16 +13,29 @@ const utils = require('../../utils/utils');
 const WebSocket = require('ws');
 const WsParser = require('simples/lib/parsers/ws');
 
+const hex = (n) => {
+  let _n = n.toString(16);
+  return _n.length === 1 ? '0' + _n : _n;
+};
+
 const parseJsonMaybe = (string) => {
   let object;
   try {
-    if (string && typeof string === 'string') object = JSON.parse(string);
+    if (typeof string === 'string') {
+      if (string) {
+        object = JSON.parse(string);
```

Comment:
Are we sure that this is a JSON string? If it isn't, it will crash. Might need a `try/catch`

Reply:
It is already inside a try/catch, but there is a good opportunity to narrow the scope of the try/catch, which I'll take now.